support/scripts/schemas.ts

```diff
@@ -63,6 +63,18 @@ async function main() {
   });
 
   writeToFile("./languages/csharp/Bitwarden.Sdk/schemas.cs", csharp.lines);
+  
+  writeToFile("./languages/csharp/schemas.cs", csharp.lines);
+
+  const go = await quicktype({
+    inputData,
+    lang: "go",
+    rendererOptions: {
+      package: "sdk",
+    },
```

Comment:
It seems we're only using the types and not the unmarshal/marshal helpers?

If so we should change this to only generate types and packages https://github.com/glideapps/quicktype/blob/master/packages/quicktype-core/src/language/Golang.ts#L25

Reply:
Added